lbz/resource.py

```diff
@@ -99,6 +103,12 @@ def post_request_hook(self):
         Place to configure post request hooks.
         """
 
+    @staticmethod
```

Comment:
it could be a good idea to make it `abstractmethod`

Reply:
then You would need to always have it this makes it stll optional